package.json

```diff
@@ -31,17 +31,22 @@
     {
       "name": "protazy",
       "url": "https://github.com/protazy"
+    },
+    {
+      "name": "footballencarta",
+      "url": "https://github.com/footballencarta"
     }
   ],
+  "dependencies": {
+    "jasmine-core": ">=3.2"
```

Comment:
Isn't `peerDependencies` the right place to have this though? This package is a plugin, so typically you want all users of this package to explicitly have `jasmine` also in their `package.json`, rather than letting it be pulled in as a dependency of the plugin.